README.md

```diff
@@ -131,6 +131,73 @@ PROS (+)
 CONS (-)
   * Problem: Managed Service Constraints. You trade control for convenience. You have limited access to the underlying OS and cannot fine-tune every low-level database or system parameter. You are also dependent on AWS's timeline for support of new database versions and features.
 ```
+## Dojo Session
+
+### Livekit VS MediaSoup
```

Comment:
Could you update the tradeoff using this template? We should not put both  Livekit and MediaSoup tradeoffs together.

**Livekit VS MediaSoup

Livekit
   PROS 
    -
   CONS
    - 

MediaSoup
   PROS 
    -
   CONS
    -**